### PR TITLE
update gribi tests to use become leader

### PR DESCRIPTION
**Pull request by Ankur19**

### CODE REVIEW

File: feature/experimental/gribi/ate_tests/dut_daemon_failure/dut_daemon_failure_test.go [hunk 1]
```diff
@@ -299,16 +299,16 @@ func TestDUTDaemonFailure(t *testing.T) {
 		// Set parameters for gRIBI client clientA.
 		// Set Persistence to true.
 		clientA := &gribi.Client{
-			DUT:                  dut,
-			FibACK:               false,
-			Persistence:          true,
-			InitialElectionIDLow: 10,
+			DUT:         dut,
+			FibACK:      false,
```
Comment (robshakir) [inline feature/experimental/gribi/ate_tests/dut_daemon_failure/dut_daemon_failure_test.go:303]: (nit) Should this be `FIBACK` given that we usually capitalise acronyms like this in Go?

Comment (Ankur19) [reply feature/experimental/gribi/ate_tests/dut_daemon_failure/dut_daemon_failure_test.go]: Done!

File: feature/gribi/ate_tests/base_leader_election_test/base_leader_election_test.go [hunk 2]
```diff
@@ -288,4 +287,7 @@ func TestElectionIDChange(t *testing.T) {
 	}
 
 	testIPv4LeaderActiveChange(ctx, t, args)
+
+	// Flush all entries after test. ClientA or ClientB doesn't matter since we use Election Override on FlushAll.
+	clientA.FlushAll(t)
```
Comment (bstoll) [inline feature/gribi/ate_tests/base_leader_election_test/base_leader_election_test.go:292]: Shouldn't most uses of FlushAll be moved to the top (after clientA.Close()) as defers so they execute if the test fails too?

Comment (Ankur19) [reply feature/gribi/ate_tests/base_leader_election_test/base_leader_election_test.go]: Done!

**Review by bstoll — APPROVED** [review]

**Pull request merged**